src/labkey/query/Rows.ts

```diff
@@ -37,6 +37,24 @@ export function deleteRows(options: IQueryRequestOptions) {
     return sendRequest(options);
 }
 
+/**
+ * Delete all rows in a table.
+ * @see LABKEY.Query.ModifyRowsResults
+ * @see LABKEY.Query.ModifyRowsOptions
+ * @param {IQueryRequestOptions} options
+ * @returns {Mixed} In client-side scripts, this method will return a transaction id
+ * for the async request that can be used to cancel the request
+ * (see <a href="http://dev.sencha.com/deploy/dev/docs/?class=Ext.data.Connection&member=abort" target="_blank">Ext.data.Connection.abort</a>).
+ * In server-side scripts, this method will return the JSON response object (first parameter of the success or failure callbacks.)
+ */
+export function truncateTable(options: IQueryRequestOptions)
+{
+    if (arguments.length > 1)
```

Comment:
Since `truncateTable` was [recently exposed](https://github.com/LabKey/platform/commit/1de76fc7a7dc128045310f068b7c87ef032accb6#diff-d32b9e107bf972db4bea1a72bfb0a53eR928) to the JS API it doesn't need this backwards compatibly support for argument length. This should be changed to:

```ts
export function truncateTable(options: IQueryRequestOptions) {
    options.action = 'truncateTable.api';
    return sendRequest(options);
}
```